src/vs/editor/contrib/hover/browser/hoverAccessibleViews.ts

```diff
@@ -2,48 +2,205 @@
+		return content.join('\n');
+	}
+
+	public onOpen(): void {
+		if (!this._hoverController) {
+			return;
+		}
+		this._hoverController.shouldKeepOpenOnEditorMouseMoveOrLeave = true;
+		this._markdownHoverFocusedIndex = this._hoverController.focusedMarkdownHoverIndex();
+		this._onHoverContentsChanged = this._register(this._hoverController.onHoverContentsChanged(() => {
+			this._onDidChangeContent.fire();
+		}));
+	}
+
+	public onClose(): void {
+		if (!this._hoverController) {
+			return;
+		}
+		this._markdownHoverFocusedIndex = -1;
+		this._hoverController.focus();
+		this._hoverController.shouldKeepOpenOnEditorMouseMoveOrLeave = false;
+		this._onHoverContentsChanged?.dispose();
+	}
+
+	private _initializeActions(): void {
+		this.actions.push(this._getActionFor(HoverVerbosityAction.Increase));
+		this.actions.push(this._getActionFor(HoverVerbosityAction.Decrease));
+	}
+
+	private _getActionFor(action: HoverVerbosityAction): IAction {
+		let actionId: string;
+		let accessibleActionId: string;
+		let actionLabel: string;
+		let actionCodicon: ThemeIcon;
+		switch (action) {
+			case HoverVerbosityAction.Increase:
+				actionId = INCREASE_HOVER_VERBOSITY_ACTION_ID;
+				accessibleActionId = INCREASE_HOVER_VERBOSITY_ACCESSIBLE_ACTION_ID;
+				actionLabel = INCREASE_HOVER_VERBOSITY_ACTION_LABEL;
+				actionCodicon = Codicon.add;
+				break;
+			case HoverVerbosityAction.Decrease:
+				actionId = DECREASE_HOVER_VERBOSITY_ACTION_ID;
+				accessibleActionId = DECREASE_HOVER_VERBOSITY_ACCESSIBLE_ACTION_ID;
+				actionLabel = DECREASE_HOVER_VERBOSITY_ACTION_LABEL;
+				actionCodicon = Codicon.remove;
+				break;
+		}
+		return new Action(accessibleActionId, actionLabel, ThemeIcon.asClassName(actionCodicon), true, () => {
+			this._editor.getAction(actionId)?.run({ index: this._markdownHoverFocusedIndex, focus: false });
+		});
+	}
+
+	private _descriptionsOfVerbosityActions(hoverController: HoverController): string[] {
+		const content: string[] = [];
+		const descriptionForIncreaseAction = this._descriptionOfVerbosityAction(hoverController, HoverVerbosityAction.Increase);
+		if (descriptionForIncreaseAction !== undefined) {
+			content.push(descriptionForIncreaseAction);
+		}
+		const descriptionForDecreaseAction = this._descriptionOfVerbosityAction(hoverController, HoverVerbosityAction.Decrease);
+		if (descriptionForDecreaseAction !== undefined) {
+			content.push(descriptionForDecreaseAction);
+		}
+		return content;
+	}
+
+	private _descriptionOfVerbosityAction(hoverController: HoverController, action: HoverVerbosityAction): string | undefined {
+		const isActionSupported = hoverController.doesMarkdownHoverAtIndexSupportVerbosityAction(this._markdownHoverFocusedIndex, action);
+		if (!isActionSupported) {
+			return;
+		}
+		let actionId: string;
+		let descriptionWithKb: string;
+		let descriptionWithoutKb: string;
+		switch (action) {
+			case HoverVerbosityAction.Increase:
```

Comment:
The accessible view service now resolves keybindings for the help dialogs. This is the case so that we have uniform language when there is no assigned keybinding and because we offer a configure keybinding action for commands with no keybindings. 

Here's what the chat accessibility help does for example:

https://github.com/microsoft/vscode/blob/3f44453bbc6da6fc31b04f8602a96fbcb06e64c7/src/vs/workbench/contrib/chat/browser/actions/chatAccessibilityHelp.ts#L42

Reply:
Oh I see thanks for the tip, I will implement that